runtime/compiler/env/j9method.cpp

```diff
@@ -316,7 +316,7 @@ bool TR_ResolvedJ9Method::isMethodInValidLibrary()
 void
 TR_J9MethodBase::parseSignature(TR_Memory * trMemory)
    {
-   U_8 tempArgTypes[256];
+   U_8 tempArgTypes[512];
```

Comment:
What's the limit here?  How do we know that 512 bytes is enough to cover all cases?

Reply:
I just doubled the existing limit for the failing test case to pass, which was testing some extreme number of args in methods being invoked and needed just 20 more bytes than the buffer size. I do not know what the original justification for keeping a fixed size buffer here is. I did want to have this dynamically allocated according to the argument length, which would be safer.

Reply:
OK.  Please open an issue for us to revisit a more robust solution in the near future.